docs/shell.md

```diff
@@ -6,7 +6,7 @@ It's a useful tool to quickly validate your assumptions about Spark in Scala bef
 Great! Let us test out our spark-shell setup with the official Pi example from the Spark distribution.
 
 ```shell
-$ spark-client.spark-shell
+$ spark-client.shell --conf spark.driver.host={LOCALHOST_IP}
```

Comment:
This localhost ip business, should we take care of it like we do for k8s master auto detection.

Reply:
yes, that a good point, but let's leave it as you suggest as next improvement